<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Research | Jin Jin</title> <meta name="author" content="Jin Jin"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%90%B3&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://jin93.github.io/research/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body> <p> ... <script async src="https://www.googletagmanager.com/gtag/js?id=G-1WXLM71CRX"></script> <script>function gtag(){dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-1WXLM71CRX");</script> </p> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Jin </span>Jin</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">Bio</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">CV</a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">People</a> </li> <li class="nav-item "> <a class="nav-link" href="/positions/">Positions</a> </li> <li class="nav-item active"> <a class="nav-link" href="/research/">Research<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/software/">Software</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Research</h1> <p class="post-description"></p> </header> <article> <div class="publications"> <p>Please see <a href="https://scholar.google.com/citations?user=yMMz4BMAAAAJ&amp;hl=en" target="_blank" rel="external nofollow noopener">Google Scholar</a> for a full list of publications.</p> <p> </p> <hr> <h5 id="polygenic-risk-prediction"><strong>Polygenic Risk Prediction</strong></h5> <div class="row justify-content-sm-center"> <div class="col-sm-4 mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/MUSSEL_GA-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/MUSSEL_GA-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/MUSSEL_GA-800.webp"></source> <img src="/assets/img/MUSSEL_GA.jpg" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="Example Figure: Disease Risk Prediction" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="col-sm-8 mt-3 mt-md-0">While polygenic risk scores (PRS) become increasingly robust with increasingly larger GWAS, there are considerable debates about its clinical utility. We develop enhanced algorithms and software tools for improving the predictive utility and transferability of polygenic risk prediction models across diverse populations and complex human traits.</div> </div> <ul> <li> <strong>Jin, J.</strong>, Zhan, J., Zhang, J., Zhao, R., O’Connell, J., Jiang, Y., Aslibekyan, S., Auton, A., Babalola, E., Bell, R.K., et al. MUSSEL: Enhanced Bayesian polygenic risk prediction leveraging information across multiple ancestry groups. <em>Cell Genomics</em>, 4(4), 100539, 2024. <a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(24)00095-8" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/MUSSEL" rel="external nofollow noopener" target="_blank">[Software]</a> </li> <li>Dun, Y., Chatterjee, N., <strong>Jin, J.</strong>\(^*\), Nishimura, A\(^*\). A Robust Bayesian Method for Building Polygenic Risk Scores using Projected Summary Statistics and Bridge Prior, <em>arXiv</em>, 2024. <a href="https://arxiv.org/abs/2401.15014" rel="external nofollow noopener" target="_blank">[PDF]</a> </li> <li>Zhang, H., Zhan, J., <strong>Jin, J.</strong>, Zhang, J., Ahearn, T., Yu, Z., O’ Connell, J., Jiang, Y., Koelsch, B., 23andMe research team, Lin, X., Garcia-Closas, M., Chatterjee, N. A New Method for Multiancestry Polygenic prediction Improves Performance across Diverse Populations. <em>Nature Genetics</em>, 55(10), 1757-1768, 2023. <a href="https://www.nature.com/articles/s41588-023-01501-z" rel="external nofollow noopener" target="_blank">[PDF]</a><br> <a href="https://github.com/andrewhaoyu/CTSLEB" rel="external nofollow noopener" target="_blank">[Software]</a> <a href="https://github.com/andrewhaoyu/multi_ethnic" rel="external nofollow noopener" target="_blank">[Code]</a> </li> <li>Zhang, J., Zhan, J., <strong>Jin, J.</strong>, Ma, C., Zhao, R., Connell, J.O., Jiang, Y., 23andMe Research Team, Koelsch, B.L., Zhang, H., Chatterjee, N. An Ensemble Penalized Regression Method for Multi-ancestry Polygenic Risk Prediction. <em>Nature Communications</em>, 15(1), 3238, 2024. <a href="https://www.nature.com/articles/s41467-024-47357-7" rel="external nofollow noopener" target="_blank">[PDF]</a><br> <a href="https://github.com/Jingning-Zhang/PROSPER" rel="external nofollow noopener" target="_blank">[Software]</a> <a href="https://github.com/Jingning-Zhang/PROSPER_analysis" rel="external nofollow noopener" target="_blank">[Code]</a> </li> <li>Yu, Z., <strong>Jin, J.</strong>, Tin, A., Kottgen, A., Yu B., Chen J., Ballantyne, C.M., Hoogeveen, R.C., Arking, D.E., Chatterjee, N., Coresh, J., Grams, M.E., Coresh J. Polygenic Risk Scores for Kidney Function to the Circulating Proteome, and Incident Kidney Diseases: the Atherosclerosis Risk in Community Study. <em>Journal of the American Society of Nephrology</em>, 2021. DOI: 10.1681/ASN.2020111599.<br> <a href="https://jasn.asnjournals.org/content/jnephrol/early/2021/11/02/ASN.2020111599.full.pdf?with-ds=yes" rel="external nofollow noopener" target="_blank">[PDF]</a><br> (This work was selected by the American Society of Nephrology (ASN) as the “Best of ASN Journals” in 2021.)</li> <li>Rabinowitz, J.A., Jin, J., Kahn, G., Kuo, S.I., Campos, A., Renteria, M., Benke, K., Wilcox, H., Ialongo, N.S., Maher, B.S. and others. Genetic Propensity for Risky Behavior and Depression and Risk of Lifetime Suicide Attempt among Urban African Americans in Adolescence and Young Adulthood. <em>American Journal of Medical Genetics Part B: Neuropsychiatric Genetics</em>, 2021. DOI: 10.1002/ajmg.b.32866. <a href="https://onlinelibrary.wiley.com/doi/10.1002/ajmg.b.32866" rel="external nofollow noopener" target="_blank">[PDF]</a> </li> <li>Rabinowitz, J., <strong>Jin, J.</strong>, Kuo, S., Thrul, J., Reboussin, B., Domingue, B., Ialongo, N., Maher, B.,Uhl, G. Positive Associations between Cannabis and Alcohol Use Polygenic Risk Scores and Phenotypic Opioid Misuse among African-Americans. <em>PLOS One</em>, 17(4), p.e0266384, 2022. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0266384" rel="external nofollow noopener" target="_blank">[PDF]</a> </li> </ul> <p> </p> <hr> <h5 id="integrative-risk-prediction-models-for-human-diseases"><strong>Integrative risk prediction models for human diseases</strong></h5> <div class="row justify-content-sm-center"> <div class="col-sm-5 mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/risk_prediction_long-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/risk_prediction_long-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/risk_prediction_long-800.webp"></source> <img src="/assets/img/risk_prediction_long.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="Example Figure: Disease Risk Prediction" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="col-sm-7 mt-3 mt-md-0">Large-scale epidemiologic studies, including modern genome-wide association studies (GWAS), are now rapidly leading to the identification of novel risk factors for human diseases. To summarize these findings into more effective strategies for reducing disease burden, a critical step is to advance risk prediction tools that predict future risk of diseases for healthy individuals and precision medicine to guide treatment strategies for targeted groups of patients.</div> </div> <ul> <li> <strong>Jin, J.\(^*\)</strong>, Agarwala, N.\(^*\), Kundu, P.\(^*\), Harvey, B., Zhang, Y., Wallace, E., , Chatterjee, N. Individual and Community-level Risk for COVID-19 Mortality in the United States. <em>Nature Medicine</em>, 27(2), 264-269, 2021. <a href="https://www.nature.com/articles/s41591-020-01191-8.pdf" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/COVID19Risk" rel="external nofollow noopener" target="_blank">[Code]</a> <a href="http://covid19risktools.com/" rel="external nofollow noopener" target="_blank">[Online Tools]</a> </li> <li>Ballreich, J., <strong>Jin, J.</strong>, Kundu, P., Chatterjee, N. Provider and Patient Characteristics of Medicare Beneficiaries Who Are High-Risk for COVID-19 Mortality, <em>Journal of General Internal Medicine</em>, 36(7), 2189–2190, 2021. <a href="https://link.springer.com/article/10.1007/s11606-021-06857-8" rel="external nofollow noopener" target="_blank">[PDF]</a> </li> </ul> <h2 id="section"> </h2> <h5 id="high-dimensional-tests-for-disease-associated-gene-pathways"><strong>High-dimensional tests for disease-associated gene pathways</strong></h5> <div class="row justify-content-sm-center"> <div class="col-sm-5 mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/T2DAG_long-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/T2DAG_long-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/T2DAG_long-800.webp"></source> <img src="/assets/img/T2DAG_long.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="T2DAG" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="col-sm-7 mt-3 mt-md-3">A major task in genetic studies is to identify genes related to human traits/diseases. Compared to marginal analyses of individual genes, detecting gene pathway, i.e., gene sets with known interactions that collectively contribute to some biological functions, can provide more biologically meaningful results. Due to the typically limited sample sizes, such analyses are usually high-dimensional, where the existing tests tend to have compromised power since they do not or only inefficiently incorporate the external pathway information on gene interactions. </div> </div> <p>I collaborated with Dr. <a href="https://taryue.github.io/" rel="external nofollow noopener" target="_blank">Yue Wang</a> from Arizona State Univeristy and proposed a graph-informed test which efficiently leverages the auxiliary pathway information via structural equation modeling to improve the estimation of the precision matrix.</p> <ul> <li> <strong>Jin, J.</strong>, Yue, W. T2-DAG: A Powerful Test for Differentially Expressed Gene Pathways via Graph-informed Structural Equation Modeling. <em>Bioinformatics</em>, btab770, 2021. <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btab770/6424893?guestAccessKey=7cc27d27-ae5c-49e7-aad0-a5111fc6ac92" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/T2DAG" rel="external nofollow noopener" target="_blank">[Code]</a> </li> </ul> <p> </p> <hr> <h5 id="mendelian-randomization-mr-analysis-for-latent-exposures"><strong>Mendelian randomization (MR) analysis for latent exposures</strong></h5> <div class="row justify-content-sm-center"> <div class="col-sm-5 mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/MRLE-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/MRLE-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/MRLE-800.webp"></source> <img src="/assets/img/MRLE.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="MRLE" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="col-sm-7 mt-3 mt-md-0">Recent MR research has focused on improving robustness to the presence of pleiotropic associations, by which genetic instruments can affect the outcome independent of the exposure and thus leading to the violation of key assumptions. I investigated a novel setting of pleiotropic association where genetic variants might be associated with multiple observed traits through an underlying latent exposure which may have a causal effect on the outcome.</div> </div> <p> </p> <ul> <li> <strong>Jin, J.</strong>, Qi, G., Yu, Z., Chatterjee, N. Mendelian Randomization Analysis Using Multiple Biomarkers of an Underlying Common Exposure. Under Revision. <a href="https://doi.org/10.1101/2021.02.05.429979" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/MRLE" rel="external nofollow noopener" target="_blank">[Code]</a> <a href="https://drive.google.com/file/d/1VJJbCpMMm7qld0f0_qB9n32d4L6Om4XN/view?usp=sharing" rel="external nofollow noopener" target="_blank">[Slide]</a> </li> </ul> <p> </p> <hr> <h5 id="mpmri-based-classification-of-prostate-cancer-pca"><strong>MpMRI-based classification of Prostate cancer (PCa)</strong></h5> <div class="row justify-content-sm-center"> <div class="col-sm-5 mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/PCA_long-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/PCA_long-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/PCA_long-800.webp"></source> <img src="/assets/img/PCA_long.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="PCa detection" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="col-sm-7 mt-3 mt-md-0">My research during the Ph.D. program focused on voxel-wise PCa detection using mpMRI, a critical tool in PCa diagnosis and management. MpMRI examinations are currently interpreted manually which is limited due to inter-reader variability. Automatic, quantitative predictive methods are thus proposed to address these limitations. We identified critical issues of the existing methods and developed scalable, high-field MRI-based Bayesian and machine learning-based classifers that can be used as non-invasive tools to assist clinicians with PCa diagnosis and treatment.</div> </div> <ul> <li> <strong>Jin, J.</strong>, Zhang, L., Leng, E., Metzger, G.J., Koopmeiners, J.S. Bayesian Spatial Models for Voxel-wise Prostate Cancer Classification Using Multi-parametric MRI Data. <em>Statistics in Medicine</em>, 1-17, 2021. DOI: https://doi.org/10.1002/sim.9245. <a href="https://arxiv.org/abs/2001.07316" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/Multi-Resolution-SL" rel="external nofollow noopener" target="_blank">[Code]</a> </li> <li> <strong>Jin, J.</strong>, Zhang, L., Leng, E., Metzger, G.J., Koopmeiners, J.S. A Multi-resolution Super Learner Algorithm for Voxel-wise Classification of Prostate Cancer Using Multi-parametric MRI. To appear in <em>Journal of Applied Statistics</em>, 2021. <a href="https://arxiv.org/pdf/2007.00816.pdf" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/Multi-Resolution-SL" rel="external nofollow noopener" target="_blank">[Code]</a> </li> <li>Leng, E., Spilseth, Leng, E., Henriksen, J.C., Rizzardi, A.E., <strong>Jin, J.</strong>, Nam, J.W., Brassuer, B.M., Johnson, A.D., Reder, N.P., Koopmeiners, J.S., Schmechel, S.C., Metzger, G.J. Signature Maps for Automatic Identification of Prostate Cancer from Colorimetric Analysis of H&amp;E-and IHC-stained Histopathological Specimens. <em>Scientific Reports</em>, 9(1), 1-12, 2019. <a href="https://www.nature.com/articles/s41598-019-43486-y" rel="external nofollow noopener" target="_blank">[PDF]</a> </li> <li> <strong>Jin, J.</strong>, Zhang, L., Leng, E., Metzger, G.J., Koopmeiners, J.S. Detection of Prostate Cancer with Multiparametric MRI Utilizing the Anatomic Structure of the Prostate. <em>Statistics in medicine</em>, 37(22), 3214-3229, 2018. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/sim.7810" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/PreliminaryPoC" rel="external nofollow noopener" target="_blank">[Code]</a> </li> <li>Leng, E., Spilseth, B., Zhang, L., <strong>Jin, J.</strong>, Koopmeiners, J.S., Metzger, G.J. Development of A Measure for Evaluating Lesion-wise Performance of CAD Algorithms in the Context of MpMRI Detection of Prostate Cancer. <em>Medical Physics</em>, 45(5), 2076-2088, 2018. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6734092/" rel="external nofollow noopener" target="_blank">[PDF]</a> </li> </ul> <p> </p> <hr> <h5 id="early-phase-oncology-trial-design"><strong>Early-phase oncology trial design</strong></h5> <div class="row justify-content-sm-center"> <div class="col-sm-5 mt-3 mt-md-0"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/CBHM_long-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/CBHM_long-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/CBHM_long-800.webp"></source> <img src="/assets/img/CBHM_long.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" title="Basket Trials" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="col-sm-7 mt-3 mt-md-0">As oncology research are focusing more on the genomic aberrations potentially shared by multiple cancer types, basket trials have been proposed to assess the efficacy of a new treatment simultaneously on multiple cancer types. But even with shared genetic aberration, different cancers may respond to treatments differently. I collaborated with a research team at Sanofi to develop novel Bayesian methods for detecting treatment efficacy and proof of concept (PoC) in early-phase basket trials by flexible borrowing according to sample similarity measured by the distance between posterior distributions of the treatment effects.</div> </div> <ul> <li> <strong>Jin, J.</strong>, Riviere, M.K., Luo, X., Dong, Y. Bayesian Methods for the Analysis of Early-phase Oncology Basket Trials with Information Borrowing across Cancer Types. <em>Statistics in Medicine</em>. 39(25), 3459-3475, 2020. <a href="https://doi.org/10.1002/sim.8675" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/CBHM" rel="external nofollow noopener" target="_blank">[Code]</a> </li> <li> <strong>Jin, J.</strong>, Liu, Q., Zheng, W., Shun, Z., Lin, T.T., Gao, L., Dong, Y. A Bayesian Method for the Detection of Proof of Concept in Early Phase Oncology Studies with a Basket Design. <em>Statistics in Biosciences</em>. 12, 167-179, 2020. <a href="https://link.springer.com/article/10.1007/s12561-020-09267-2" rel="external nofollow noopener" target="_blank">[PDF]</a> <a href="https://github.com/Jin93/PreliminaryPoC" rel="external nofollow noopener" target="_blank">[Code]</a> </li> </ul> <p> </p> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Jin Jin. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>